buổi-10/main.js

```diff
@@ -0,0 +1,150 @@
+// bài 1
+const getMultiplication = number => {
+    if (number >= 1 && number <= 10) {
+        for(let i = 1; i <= 10; i++) {
+            console.log(number + ' * ' + i + ' = ' + (number * i))
+        } 
+    } else {
+        console.log('Chỉ nhận giá trị từ 1-> 10!!')
+    }
+}
+
+// bài 2
```

Comment:
```suggestion
// Bài 2
```
Mọi thứ đều phải dùng tiếng anh, ngoại trừ được yêu cầu dùng ngôn ngữ khác, hoặc bắt buộc phải dùng ngôn ngữ khác